analyzers/tests/SonarAnalyzer.UnitTest/Rules/MemberShouldBeStaticTest.cs

```diff
@@ -30,50 +30,51 @@ namespace SonarAnalyzer.UnitTest.Rules
     [TestClass]
     public class MemberShouldBeStaticTest
     {
+        private readonly VerifierBuilder builder = new VerifierBuilder<MemberShouldBeStatic>();
+
         [DataTestMethod]
         [DataRow("1.0.0", "3.0.20105.1")]
         [DataRow(Constants.NuGetLatestVersion, Constants.NuGetLatestVersion)]
         public void MemberShouldBeStatic(string aspnetCoreVersion, string aspnetVersion) =>
-            OldVerifier.VerifyAnalyzer(@"TestCases\MemberShouldBeStatic.cs",
-                                    new MemberShouldBeStatic(),
-                                    NuGetMetadataReference.MicrosoftAspNetCoreMvcWebApiCompatShim(aspnetCoreVersion)
+            builder.AddPaths("MemberShouldBeStatic.cs")
+                .AddReferences(NuGetMetadataReference.MicrosoftAspNetCoreMvcWebApiCompatShim(aspnetCoreVersion)
                                         .Concat(NuGetMetadataReference.MicrosoftAspNetMvc(aspnetVersion))
                                         .Concat(NuGetMetadataReference.MicrosoftAspNetCoreMvcCore(aspnetCoreVersion))
                                         .Concat(NuGetMetadataReference.MicrosoftAspNetCoreMvcViewFeatures(aspnetCoreVersion))
                                         .Concat(NuGetMetadataReference.MicrosoftAspNetCoreRoutingAbstractions(aspnetCoreVersion))
-                                        .ToImmutableArray());
+                                        .ToImmutableArray())
+                .Verify();
 
 #if NET
         [TestMethod]
         public void MemberShouldBeStatic_CSharp9() =>
-            OldVerifier.VerifyAnalyzerFromCSharp9Console(@"TestCases\MemberShouldBeStatic.CSharp9.cs", new MemberShouldBeStatic());
+            builder.AddPaths("MemberShouldBeStatic.CSharp9.cs").WithTopLevelStatements().Verify();
 #endif
 
         [TestMethod]
         public void MemberShouldBeStatic_CSharp8() =>
-            OldVerifier.VerifyAnalyzer(@"TestCases\MemberShouldBeStatic.CSharp8.cs",
-                new MemberShouldBeStatic(),
-                ParseOptionsHelper.FromCSharp8,
-                MetadataReferenceFacade.NETStandard21);
+            builder.AddPaths("MemberShouldBeStatic.CSharp8.cs")
+                .WithOptions(parseOptions: ParseOptionsHelper.FromCSharp8)
```

Comment:
We can remove `parseOptions:` here since the parameter type is clear from the method name and also from the parameter.